async function run() {}

run()
  .then((val) => {
    console.log(val);
  })
  .catch((err) => console.error(err));
